docs/deploy-with-docker.md

```diff
@@ -19,7 +19,9 @@
 
 # Building and Launching with Docker
 
-To build a Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
+**NOTE:** We support official Docker Image, please visit [DockerHub](https://hub.docker.com/r/apache/apisix-dashboard) for more information. The following steps are for building Docker Image manually.
+
+To build the Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
```

Comment:
How about provide a download link for this `Dockerfile`?

Reply:
🤔 We may use this repo[1] to manage all products' Dockerfile in M2.4, let me create an issue to track this.

[1] https://github.com/apache/apisix-docker

Reply:
https://github.com/apache/apisix-dashboard/issues/1265